product_docs/docs/biganimal/release/using_cluster/01_postgres_access/pg_ba_admin.mdx

```diff
@@ -0,0 +1,73 @@
+| pg_read_all_data     | Read all data (tables, views, sequences), as if having SELECT rights on those objects, and USAGE rights on all schemas, even without having it explicitly. This role doesn't have the role attribute BYPASSRLS set. If RLS is being used, an administrator may wish to set BYPASSRLS on roles which this role is GRANTed to.                      |
+| pg_write_all_data    | Write all data (tables, views, sequences), as if having INSERT, UPDATE, and DELETE rights on those objects, and USAGE rights on all schemas, even without having it explicitly. This role does not have the role attribute BYPASSRLS set. If RLS is being used, an administrator may wish to set BYPASSRLS on roles which this role is GRANTed to. |
+| pg_read_all_settings | Read all configuration variables, even those normally visible only to superusers.                                                                                                                                                                                                                                                                  |
+| pg_read_all_stats    | Read all pg_stat_* views and use various statistics related extensions, even those normally visible only to superusers.                                                                                                                                                                                                                            |
+| pg_stat_scan_tables  | Execute monitoring functions that may take ACCESS SHARE locks on tables, potentially for a long time.                                                                                                                                                                                                                                              |
+| pg_monitor           | Read/execute various monitoring views and functions. This role is a member of pg_read_all_settings, pg_read_all_stats and pg_stat_scan_tables.                                                                                                                                                                                                     |
+| pg_database_owner    | None. Membership consists, implicitly, of the current database owner.                                                                                                                                                                                                                                                                              |
+
+## Predefined roles not assigned to pg_ba_admin
+
+A user with the pg_ba_admin role doesn't has the following roles that are predefined in Postgres:
```

Comment:
We need to document that these roles can't be granted by `edb_admin` either.

Reply:
done